### PR TITLE
Fix signatures

### PR DESCRIPTION
**Pull request by OpatrilPeter**
Several changes in function signatures for better clarity.

### CODE REVIEW

File: include/superior_mysqlpp/low_level/dbdriver.hpp [hunk 1]
```diff
@@ -130,6 +127,11 @@ namespace SuperiorMySqlpp { namespace LowLevel
         DBDriver(DBDriver&&) = default;
         DBDriver& operator=(DBDriver&&) = delete;
 
+        /**
+         * Type for indexing rows.
+         * Underlying type inferred from usage in C MySQL client.
+         */
+        using RowIndex_t = unsigned long long;
```
Comment (smradCZ) [inline include/superior_mysqlpp/low_level/dbdriver.hpp]: There isn't needed to append `_t` to typename.

File: include/superior_mysqlpp/low_level/dbdriver.hpp [hunk 2]
```diff
@@ -70,9 +70,6 @@ namespace SuperiorMySqlpp { namespace LowLevel
         const std::uint_fast64_t id;
         MYSQL mysql;
         Loggers::SharedPointer_t loggerPtr;
-
-        using size_t = unsigned long long;
```
Comment (smradCZ) [inline include/superior_mysqlpp/low_level/dbdriver.hpp:108]: This is API breaking change, it should be mentioned in changelog at least.

**Review by smradCZ — CHANGES_REQUESTED** [review]

File: include/superior_mysqlpp/low_level/dbdriver.hpp (hunk 2 above)
Comment (OpatrilPeter) [reply include/superior_mysqlpp/low_level/dbdriver.hpp:108]: This definition is in private section.

**Review by smradCZ — APPROVED** [review]

**Comment by OpatrilPeter** [comment]
I was revising open PRs and accidentaly closed this one. It was approved, but we internally talked about more extensive rewrite and put this on hold for that reason. The rewrite never materialized, so I'll go ahead and try wrapping up this one.

**Pull request closed without merging**